Document command-line flag for delaying enrollment (#1375)
elastic/observability-docs: docs/en/ingest-management/commands.asciidoc

`--certificate-authorities <string>`::
Comma-separated list of root certificates used for server verification.

`--delay-enroll`::
Delays enrollment to occur on first start of the {agent} service. This setting

> is useful when there is no ability to run {agent} at installation time, for

Reviewer: ```suggestion
is useful when the install doesn't want the {agent} to start until next reboot or manual start of the service, for
```

Author: Made a couple of minor edits to this statement to remove anthropomorphism (the install can't want things). Hope that's OK! :-)

> is useful when you want the {agent} to start after the next reboot or manual start of the service, for

Author: ```suggestion
is useful when you don't want the {agent} to start until after the next reboot or manual start of the service, for
```

> is useful when you don't want the {agent} to start until after the next reboot or manual start of the service, for

Author: ```suggestion
is useful when you don't want the {agent} to enroll until after the next reboot or manual start of the service, for
```
